resources/options.js

```diff
@@ -19,20 +28,18 @@ function changeListener() {
 function saveOptions() {
     chrome.storage.sync.set({
         // General
-        loggingEnabled: $('#logging').prop('checked'),
+        loggingEnabled: $(optionsID(0)).prop('checked'),
         // Privacy
-        trackTabs: $('#tabs').prop('checked'),
-        trackComments:  $('#comments').prop('checked'),
-        trackPeerComments: $('#peerComments').prop('checked'),
-        trackFocus: $('#focus').prop('checked'),
+        trackTabs: $(optionsID(1)).prop('checked'),
+        trackComments:  $(optionsID(2)).prop('checked'),
+        trackPeerComments: $(optionsID(3)).prop('checked'),
+        trackFocus: $(optionsID(4)).prop('checked'),
         // Security
-        hashUsername: $('#username').prop('checked'),
-        hashRepo: $('#repo').prop('checked'),
-        hashFile: $('#file').prop('checked')
+        hashUsername: $(optionsID(5)).prop('checked'),
+        hashRepo: $(optionsID(6)).prop('checked'),
+        hashFile: $(optionsID(7)).prop('checked')
         // Hints
-    }, function() {
-        Materialize.toast("Options saved!", 2000);
-    });
+    }, function() {});
```

Comment:
I've changed the naming of the option names in the array. Because of this, it was possible to implement your suggestion, which saves us a lot of LOC for the same functionality, indeed!
Check the last commit. Thanks for the suggestion! :)